ClearURLs for uBo/clear_urls_uboified.txt

```diff
@@ -1,12 +1,13 @@
-! Title: ClearURLs for uBo - List extension
+! Title: ClearURLs for uBo
 ! Homepage: https://github.com/DandelionSprout/adfilt/discussions/163
-! Description: Want to use ClearURLs without installing another extension? This list is a (unofficial) version of the ClearURLs rules, designed for use in uBlock Origin and AdGuard
-! Last updated: 02/12/2021
-! Script last updated: 21/11/2021
+! Description: Want to use ClearURLs's tracking protection without installing another extension? This list is a (unofficial) version of the ClearURLs rules, designed for use in uBlock Origin and AdGuard
```

Comment:
No idea if the grammar here is correct

Reply:
`ClearURLs's` could probably be changed to `ClearURLs'`. Otherwise it looks good enough.

Reply:
Ah, now I remember that from Language class :)
Thanks